{
    "label": {
        "command_console": "Console de comandos selecionado."
    },
    "feedback": {
        "common": {
            "entity_type": {
                "Classifier": "classificador",
                "Attribute": "atributo",
                "Method": "método",
                "Parameter": "parâmetro"
            },
            "classifier_type": {
                "class": "classe",
                "abstract": "classe abstrata",
                "interface": "interface"
            }
        },
        "create": {
            "classifier": {
                "part_1": "Criação da ",
                "part_2": " foi realizada com sucesso.",
                "error": {
                    "classifier_name_already_in_use": "Nome fornecido para classificador já está em uso.",
                    "attribute_name_already_in_use": {
                        "part_1": "Há dois ou mais atributos usando o nome ",
                        "part_2": "."
                    },
                    "syntax_error_in_method_argument": {
                        "part_1": "O seguinte argumento de método possui um erro de sintaxe: ",
                        "part_2": "."
                    },
                    "method_name_already_in_use": {
                        "part_1": "Há dois ou mais métodos usando o nome ",
                        "part_2": "."
                    }
                }
            },
            "attribute": {
                "error": {
                    "invalid_attribute_arguments": {
                        "part_1": "O seguinte atributo possui muitos ou poucos argumentos ",
                        "part_2": ", use 2 ou 3 argumentos."
                    }
                }
            },
            "parameter": {
                "error": {
                    "invalid_parameter_arguments": {
                        "part_1": "O seguinte parâmetro possui muitos ou poucos argumentos ",
                        "part_2": ", use 2 argumentos."
                    }
                }
            },
            "error": {
                "entity_type_missing_on_creation": "Nenhum tipo foi fornecido para o comando create.",
                "unrecognized_entity_type": {
                    "part1": "O tipo de entidade ",
                    "part2": " não foi reconhecido pela aplicação."
                },
                "missing_name_for_entity": {
                    "part_1": "Está faltando o nome para criação do ",
                    "part_2": "."
                },
                "missing_type_for_entity": {
                    "part_1": "Está faltando o tipo para criação do ",
                    "part_2": "."
                }
            }
        },
        "error": {
            "unrecognized_command": {
                "part_1": "O comando ",
                "part_2": " não foi reconhecido pela aplicação."
            },
            "invalid_characters_used_for_naming": {
                "part_1": "O seguinte argumento ",
                "part_2": " possui caracteres inválidos."
            },
            "no_end_given_for_argument": {
                "part_1": "Não foi provido um final para o argumento ",
                "part_2": "."
            },
            "invalid_visibility": {
                "part_1": "A visiblidade ",
                "part_2": " não é valida, use as opções public, protected ou private."
            }
        }
    },
    "error": {
        "unhandled_error": "Um erro inesperado ocorreu!",
        "unknown_error_thrown": "Erro desconhecido foi lançado!"
    }
}